src/main.ts

```diff
@@ -62,6 +75,12 @@ const command: Command = {
 			default: false
 		});
 
+		options('i', {
```

Comment:
This seems confusing alongside the `config` option. I would have thought the former would specify the config to run with. As an aside though, do we think we should be allowing the config to be specified? The CLI commands are designed to be opinionated. If the user wishes to change things to this degree then I imagine that they should `eject`.